packages/react-hooks/src/inbox/use-inbox-actions.ts

```diff
@@ -56,11 +66,29 @@ const useInboxActions = (): IInboxActions => {
 
   const events = Events({ client: courierClient });
   const messages = Messages({ client: courierClient });
+  const initialState = InitialState({ client: courierClient });
 
   return {
-    init: (payload) => {
+    init: async (payload) => {
       dispatch(initInbox(payload));
 
+      const response = await initialState.getInitialState({
```

Comment:
if someone was only to use the hooks... i feel like we maybe shouldn't fetch initial state right?  we should only do this initial state stuff if we are using the components?

Reply:
jk this "init" action only gets fired if you are using the components so it should be fine